src/modelbench/consistency_checker.py

```diff
@@ -247,6 +247,17 @@ def __init__(self, search_engine: JournalSearch, sut, test):
         self.sut = sut
         self.test = test
 
+        # security benchmarks have different criteria
+        starting_run_entry = search_engine.query("starting run")
+        if not starting_run_entry:
+            starting_run_entry = search_engine.query("starting calibration run")
+        if starting_run_entry:
+            benchmark = starting_run_entry[0]["benchmarks"][0]
+            self.benchmark = "security" if benchmark.lower().startswith("security") else "general"
+        else:
+            self.benchmark = "general"
+
+        # TODO: This assumes that all tests use the same annotators! Which
```

Comment:
Not sure if this interface is not showing me the whole comment, but the last word I see is "Which". So this doesn't look like a TODO to me. Also, I prefer that TODOs not get checked in unless they're going to be done in very short order. Otherwise they're effectively just notes, and should be phrased as such.